src/REPLMode/command_declarations.jl

```diff
@@ -340,7 +340,7 @@ reached from any existing project.
     :api => API.undo,
     :description => "undo the latest change to the active project",
     :help => md"""
-    undo
+    [u|undo]
```

Comment:
It's a bit weird that "undo" and "redo" have such short aliases. It seems like they are rare enough to not really need aliases at all, let alone single-letter aliases.

Reply:
Yeah, we could remove the aliases

Reply:
So shall I remove these two aliases in this PR?

Reply:
Best to remove before documenting so, yes, that would be great.